pom.xml

```diff
@@ -39,6 +39,10 @@
   <profiles>
     <profile>
       <id>build-individual-bundles</id>
+      <properties>
```

Comment:
You should not compare Y-build with I-builds as this is what will happen if you merge this in master.

Reply:
Yes Y build change is only required for Beta branch.